File: src/main/webapp/service/wareGroupService.js
/**
 * Created by Jasmina on 30/05/2017.
 */
app.service('wareGroupService', function ($http) {
    return{
        read: function (onSuccess, onError) {
            $http.get('api/wareGroups').then(onSuccess, onError);
        },

        create: function (wareGroup, onSuccess, onError) {
            $http.post('api/wareGroups', wareGroup).then(onSuccess, onError);
        },

        update: function (wareGroup, onSuccess, onError) {
            $http.patch('api/wareGroups/' + wareGroup.id, wareGroup).then(onSuccess, onError);
        },

        delete: function (id, onSucces, onError) {
            $http.delete('api/wareGroups/' + id).then(onSucces, onError);
        }
    }
});

File: src/main/java/com/ftn/service/WareGroupService.java
package com.ftn.service;

import com.ftn.model.WareGroup;

import java.util.List;

/**
 * Created by Jasmina on 22/05/2017.
 */
public interface WareGroupService {

    List<WareGroup> read();

    WareGroup create(WareGroup wareGroup);

    WareGroup update(Long id, WareGroup wareGroup);

    void delete(Long id);
}
